notebooks/book2/10/mc_accuracy_demo.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>get rid of these warnings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>